site/_sprints/197.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Sprint 197 Details
+sprint_number: 197
+slides: https://www.slideshare.net/ManageIQ/sprint-196
```

Comment:
@Fryguy this is pointing to the wrong sprint

Reply:
@chessbyte Want a hacktoberfest PR? 😉

Reply:
Fixed in https://github.com/ManageIQ/manageiq.org/pull/1121

Reply:
forgot about Hacktoberfest!